cw-rpm.spec.inc

```diff
@@ -1,7 +1,11 @@
+# Default the License and URL if they are not already specified.
+%{!?LICENSE_VALUE: %define LICENSE_VALUE "GPLv3+ with exceptions" }
```

Comment:
I wanted a consistent naming convention. So `X_VALUE` is the value of the `X` field.